src/layers.cpp

```diff
@@ -125,29 +125,39 @@ void Layer_::move(uint8_t layer) {
 }
 
 void Layer_::on(uint8_t layer) {
-  bool wasOn = isOn(layer);
+  // If the target layer was already on, return
+  if (bitRead(LayerState, layer))
```

Comment:
Is there a reason for using `bitRead` here instead of `isOn`? I think the code would be easier to read if we used `isOn` here (and in `Layer_::off` too).

The compiler should be able to do the right thing even when using `isOn`.

Reply:
I think it's easier to read the other way around (since the next line calls `bitSet()`), but I see your point of view.